### PR TITLE
Add FAStartDate and FAEndDate

### PR DESCRIPTION
**Pull request by gigxz**
Storing these on CustomServices as well, since financial assistance services are common and we'll want to display them in a certain way.

### CODE REVIEW

File: db/warehouse/migrate/20230606184316_add_fa_amount_ranges.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class AddFaAmountRanges < ActiveRecord::Migration[6.1]
+  def change
+    # new in 2024 spec
+    add_column :Services, :FAStartDate, :date, null: true
+    add_column :Services, :FAEndDate, :date, null: true
+
+    # add to CustomServices for custom financial assistance
+    add_column :CustomServices, :FAAmount, :float, null: true
```
Comment (gigxz) [inline db/warehouse/migrate/20230606184316_add_fa_amount_ranges.rb:8]: Is `float` the right thing to use? in the Services table the type is just "numeric" but I'm not sure how to achieve that or if its desirable

**Comment by gigxz** [comment]
Should I add these to `HmisStructure::Service` yet?

File: db/warehouse/migrate/20230606184316_add_fa_amount_ranges.rb (hunk 1 above)
Comment (eanders) [reply db/warehouse/migrate/20230606184316_add_fa_amount_ranges.rb:8]: yeah, I believe they are supposed to have cents.

**Review by eanders — APPROVED** [review]

**Comment by eanders** [comment]
I think we should only add them to the structure file when we get to the 2024 iteration (that's where that's coming from right?)

**Comment by gigxz** [comment]
Ok. Yes, it's 2024.

**Pull request merged**